definitions/TerrainTypeSounds.dbd

```diff
@@ -9,3 +9,5 @@ LAYOUT E4923C1F
 BUILD 8.0.1.26095, 8.0.1.26175, 8.0.1.26231
 $noninlineid$ID<32>
 Name
+
+BUILD 0.5.3.3368
```

Comment:
uh, empty?!

Reply:
This should be grouped into 6.0.1 and 4.0.0

Reply:
Done